src/main/java/io/confluent/connect/jdbc/source/DataConverter.java

```diff
@@ -72,8 +74,14 @@ public static Struct convertRecord(Schema schema, ResultSet resultSet, boolean m
     return struct;
   }
 
+  private static String whitelistDefaultValue(String defaultValue) {
```

Comment:
We need to figure out what the values for the actual functional defaults we need to handle are. DATE.NOW or whatever.

Reply:
In rails (fadmin), when a column is defined as having default time.now(), the mysql table has no default and ActiveRecord will populate the field at insert time before the record reaches the db

Reply:
Ah. Well, we'll still need to worry about this for applications that actually use their databases as databases, I guess.